src/pages/dashboardTab/store/dashboardReducer.js

```diff
@@ -969,14 +1055,10 @@ const reducers = {
     isDashboardTableLoading: false,
   }),
   TOGGGLE_CHECKBOX_WITH_API: (state, item) => {
-    let updatedCheckboxData1 = updateFilteredAPIDataIntoCheckBoxData(
+    const updatedCheckboxData1 = updateFilteredAPIDataIntoCheckBoxData(
```

Comment:
Codacy has a fix for the issue: [Unexpected newline after '('.](https://app.codacy.com/gh/CBIIT/bento-frontend/pullRequest?prid=8502143)
```suggestion
    const updatedCheckboxData1 = updateFilteredAPIDataIntoCheckBoxData(item.data.searchSubjects, facetSearchData,
```